test/typecheck/bad-select-value0.txt

```diff
@@ -8,10 +8,10 @@
     select
     drop))
 (;; STDERR ;;;
-out/test/typecheck/bad-select-value0.txt:8:5: error: type mismatch in select, expected [f64, f64, i32] but got [i32, f64, f32]
+out/test/typecheck/bad-select-value0.txt:8:5: error: type mismatch in select, expected [any, any, i32] but got [i32, f64, f32]
     select
     ^^^^^^
-out/test/typecheck/bad-select-value0.txt:8:5: error: type mismatch in function, expected [] but got [f64]
+out/test/typecheck/bad-select-value0.txt:8:5: error: type mismatch in function, expected [] but got [any]
```

Comment:
This seems odd because the code moves away from `Any` and towards `Void` but these test results seems to move towards "any"   .. are `Void` and the string "any" somehow releated?

Reply:
Out of curiosity do you know why did the old code includes `f64` here?

Reply:
Oops, I forgot to change the text parser too. This is back to the old output now.